src/Core/ObjectRegistry.php

```diff
@@ -40,15 +40,15 @@
  * @see \Cake\View\HelperRegistry
  * @see \Cake\Console\TaskRegistry
  * @template TObject of object
- * @template-implements \IteratorAggregate<string, TObject>
+ * @template-implements \IteratorAggregate<TObject>
  */
 abstract class ObjectRegistry implements Countable, IteratorAggregate
 {
     /**
      * Map of loaded objects.
      *
-     * @var array<string, object>
-     * @psalm-var array<string, TObject>
+     * @var array<object>
```

Comment:
We use int keys here?

Reply:
There are tests where a logger name is `404` and since it's numeric PHP converted the key to int.